src/components/mdRatingBar/mdRatingBar.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../core/stylesheets/variables.scss';
+
+$button-radius: 2px;
+
+.md-rating-bar {
+  width: auto;
+  display: flex;
+  width: fit-content;
+
+  > .front-stars {
+    overflow-x: hidden;
+    display: inherit;
+  }
+
+  > .back-stars, > .front-stars {
+    > .md-icon {
+      margin: 0;
+      white-space: nowrap;
+      cursor: pointer;
+    }
+  }
+
+  &:not([disabled]) {
```

Comment:
Well, I needed to differentiate a disabled rating from an enabled one, so I used this hover to signal this…